web/src/compose_actions.js

```diff
@@ -90,13 +90,14 @@ function clear_box() {
     $(".compose_control_button_container:has(.add-poll)").removeClass("disabled-on-hover");
 }
 
+let autosize_callback_opts;
 export function autosize_message_content(opts) {
     if (!compose_ui.is_full_size()) {
-        autosize($("textarea#compose-textarea"), {
-            callback() {
-                maybe_scroll_up_selected_message(opts);
-            },
+        autosize_callback_opts = opts;
+        $("textarea#compose-textarea").one("autosize:resized", () => {
```

Comment:
[`.one`](https://api.jquery.com/one/) is certainly wrong—remember, that means **one event**, not one handler.

Reply:
Oh shoot, I thought it was one handler. Is there a way to set up an event where it guarantees one handler? If not, I guess I should add a `compose_actions.initialize` function or something?

Reply:
For what it's worth, I did test it with a console statement in the callback and I was seeing multiple callbacks triggered-- do you know why that was?

Reply:
One pattern that one sometimes sees is do `.off().on()`.

Reply:
Actually, maybe using `one()` makes sense here? 

`autosize($("textarea#compose-textarea"))` is also called from `make_compose_box_original_size`, and I don't think it makes sense to assume the `let autosize_callback_opts` would be the same when it's called there. To make it functionally the same as a callback to this one call of `autosize`, I'd expect to set up a `.one()` each time.

Reply:
Yeah `.off.one` for one event is I think what we want for this code path.

Reply:
Done :)